src/sage/arith/misc.py

```diff
@@ -3575,14 +3600,26 @@ def CRT_list(values, moduli):
         sage: ms
         [5, 7, 9]
     """
-    if not isinstance(values, list) or not isinstance(moduli, list):
+    if not isinstance(values, list) or not isinstance(moduli, (list, type(None))):
```

Comment:
```suggestion
    if not isinstance(values, list) or (moduli is not None and not isinstance(moduli, list)):
```
This is the standard way in Python as I understand it..

Reply:
Existing code does seem to do that. There's also `from types import NoneType`, but none of the existing code does that.